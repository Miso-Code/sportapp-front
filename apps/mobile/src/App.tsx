import React from 'react'

import { SafeAreaView, Text } from 'react-native'

import { Colors } from 'react-native/Libraries/NewAppScreen'

function App(): React.JSX.Element {
	const backgroundStyle = {
		backgroundColor: Colors.lighter
	}

	return (
		<SafeAreaView style={backgroundStyle}>
			<Text>Mobile App</Text>
		</SafeAreaView>
	)
}

export default App
