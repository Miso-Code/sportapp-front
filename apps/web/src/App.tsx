import { useTranslation } from 'react-i18next'
import './App.scss'

function App() {
	const { t } = useTranslation()

	return (
		<>
			<h1>{t('app.name')}</h1>
		</>
	)
}

export default App
