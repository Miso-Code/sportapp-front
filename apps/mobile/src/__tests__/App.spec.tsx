import 'react-native'
import React from 'react'
import App from '@/App'

import renderer, { ReactTestRenderer } from 'react-test-renderer'

describe('<App />', () => {
	let wrapper: ReactTestRenderer

	beforeEach(() => {
		wrapper = renderer.create(<App />)
	})

	afterEach(() => {
		wrapper.unmount()
	})

	it('should render correctly', () => {
		expect(wrapper).toMatchSnapshot()
	})
})
