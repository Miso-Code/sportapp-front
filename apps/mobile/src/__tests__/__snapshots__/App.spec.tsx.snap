// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#F3F3F3",
    }
  }
>
  <Text>
    Mobile App
  </Text>
</RCTSafeAreaView>
`;
